routes/likeTracker.js

```diff
@@ -71,12 +84,106 @@ router.get('/dislikes', ensureAuthenticated, (req, res) => {
   LikeTracker.findOne({ userId: req.user.id }, async (err, doc) => {
     try {
       if (err) throw err;
-      if (doc) res.send({ dislikes: doc['dislikes'] });
+      if (doc) {
+        const dislikesArray = doc['dislikes'].sort(
+          (a, b) => Date.parse(b.date) - Date.parse(a.date)
+        );
+        res.send({ dislikes: dislikesArray });
+      }
       if (!doc) res.send({ dislikes: [] });
     } catch (err) {
       console.log(err);
     }
   });
 });
 
+router.delete('/likes', ensureAuthenticated, (req, res) => {
+  LikeTracker.findOne({ userId: req.user.id }, async (err, doc) => {
+    try {
+      if (err) throw err;
+      if (doc) {
```

Comment:
these delete pathways take an object that looks like this
{"filmIds": [array of film ids as strings]}